### PR TITLE
Unit Test Case for imageStream

### PR DESCRIPTION
**Pull request by uidoyen**
Closes [RHOAIENG-2146](https://issues.redhat.com/browse/RHOAIENG-2146)
## Description

Unit Test for k8s/imageStream

## How Has This Been Tested?

`npm run test`

## Test Impact

Screenshot 2024-01-25 at 8 45 39 PM

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/api/__tests__/imageStreams.spec.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      { metadata: { name: 'item 2' } },
+    ] as ImageStreamKind[];
+
+    k8sListResourceItemsMock.mockResolvedValue(getNotebookImageStreamsMock);
+
+    const renderResult = await getNotebookImageStreams('namespace');
+
+    expect(k8sListResourceItemsMock).toHaveBeenCalledWith({
+      model: ImageStreamModel,
+      queryOptions: expect.objectContaining({
```
Comment (christianvogt) [inline frontend/src/api/__tests__/imageStreams.spec.ts:25]: Why use `objectContaining` here?

Doesn't it work as expected with a direct comparison:
```suggestion
      queryOptions: {
```

Comment (uidoyen) [reply frontend/src/api/__tests__/imageStreams.spec.ts:25]: it was a type, and Yes, It does work without `objectContaining`

**Review by christianvogt — COMMENTED** [review]
Test files need to go in the `__test__` director colocated with the file being tested.

Please move your test file to `src/api/k8s/__tests__/imageStreams.spec.ts`

**Comment by uidoyen** [comment]
> Test files need to go in the `__test__` directory co-located with the file being tested.
> 
> Please move your test file to `src/api/k8s/__tests__/imageStreams.spec.ts`

Thanks @christianvogt moved to relevant folder. I apologies for overlook.

**Pull request merged**